### ISSUE TITLE
Kspace data "OSError: Input/output error"

### ISSUE THREAD

**Issue by jhri626** [LISTatSNU/FastMRI_challenge#243]
안녕하세요 조교님,
원활한 대회 운영을 위해 힘써주셔 감사합니다
아래의 사항을 문의 드리고자 올립니다

1. GPU 서버 내 특정 ksapce data 손상 문제

서버 내에서 reconstruction 도중 계속해서 다음과 같은 에러 메세지가 발생했습니다.

OSError: [Errno 5] Can't synchronously read data (file read failed: time = Wed Jul 10 00:14:45 2024
, filename = '/home/Data/leaderboard/acc5/kspace/brain_test55.h5', file descriptor = 37, errno = 5, error message = 'Input/output error', buf = 0x7f62aa6911e0, total read size = 26664496, bytes this sub-read = 26664496, bytes actually read = 18446744073709551615, offset = 0)

확인한 결과 데이터셋 자체의 손상이 있을수도 있다는 결과를 얻어서
leaderboard/acc5/kspace 폴더 내부의 brain_test55.h5 을 교체하기를 희망합니다
혹시 제가 드라이브에 있는 원본을 로컬에 다운 받아서 올려야 하는지 아니면 원격으로 변경해주실수 있는지 여쭤봅니다.
만약 로컬에서 업로드 해야한다면 어떻게 해야하는지 간단한 사용법을 알려주시면 정말 감사하겠습니다.

(iabeng26: [sad-recognition])

**Comment by jhri626** [comment]
추가적으로 brain_test56.h5, brain_test57.h5 파일도 동일한 문제가 있는것 같습니다

**Comment by yxxshin** [comment]
안녕하세요,

말씀하신대로 brain_test{55~57}.h5 파일을 확인해 본 결과, 파일에 문제가 있는 것 같습니다. 이상하게 파일을 다운로드 할 때 저도 똑같이 Input/output error가 발생하는데, 다운로드한 파일을 열어보면 또 파일은 멀쩡하네요. 

저도 정확한 원인은 모르겠지만 우선 원본 데이터로 해당 파일들을 업로드 하겠습니다. 참가자 분께서는 권한 문제로 파일을 직접 업로드할 수 없으실 겁니다. 다만 정보화지원실에서 설정을 건드렸는지 저도 현재 root 권한으로 접속이 불가능한 상태라 문의한 상황입니다. 답변을 받는 대로 접속하여 문제의 파일들을 새로 업로드하겠습니다. 혹시 다른 파일에서도 문제가 발견되면 알려주세요.

불편을 끼쳐 드려 죄송합니다.

신연상 드림

**Comment by YunseokHan** [comment]
개인적으로 Can't synchronously read data 문구로 보았을 때 다른 프로세스가 해당 파일을 열어 놓아서 동기적으로 reading이 안되는 걸 수도 있을 것 같다는 생각이 듭니다...! 혹시 확인해 주실 수 있으실까요?

**Comment by yxxshin** [comment]
저도 h5 파일 관련하여 보통은 "Can't read data" 오류가 발생하는데 이번에는 'synchronously' 라는 단어가 포함된게 의심스러워서 찾아봤는데, 아직은 잘 모르겠네요.. `lsof` 등의 명령어로 직접 확인했을 때에는 따로 그 파일 혹은 폴더를 사용중인 프로세스는 없었습니다.

좋은 아이디어 감사합니다! 다른 분들도 언제든지 편하게 참여 혹은 지적해주셔도 됩니다 :)

**Comment by jhri626** [comment]
네 저도 lsof 로 확인한 결과 해당 파일이나 폴더가 열려있는 곳은 없었습니다. 또한 모든 터미널 종료이후에 다시 실행시켜도 동일한 오류를 얻었습니다

**Comment by yxxshin** [comment]
@jhri626 안녕하세요,

조금 전에 `leaderboard/acc5/kspace/brain_test{55~57}.h5` 의 파일을 새로 업로드 하였습니다. Jupyter notebook 상에서 해당 파일들의 'Last Modified'가 최신화 된 것을 확인해 보신 후, 문제가 해결되었는지 확인해 주시기 바랍니다.

감사합니다.

**Comment by jhri626** [comment]
네 확인하였습니다 감사합니다!
